www_src/pages/editor/editor.jsx

```diff
@@ -48,5 +48,96 @@ var Editor = React.createClass({
   }
 });
 
+var TextEditor = React.createClass({
+  mixins: [React.addons.LinkedStateMixin, Binding],
+  getInitialState: function () {
+    return {
+      fontSize: 18,
+      fontFamily: 'Roboto',
+      color: '#645839',
+      fontWeight: 'normal',
+      fontStyle: 'normal',
+      textDecoration: 'none',
+      textAlign: 'center'
+    };
+  },
+  onChange: function (id) {
+    return (e) => {
+      var state = {};
```

Comment:
I am teh blindz. Sometimes, **infrequently**, but sometimes, I wish JS had PHP's $$var concept (where {$$id: ... } would be the same as obj={}, obj[id] = ... )